playbooks/roles/airship-deploy-ucp/tasks/main.yml

```diff
@@ -84,9 +84,12 @@
     - install
 
 - name: Apply Armada cluster role binding
-  shell: "kubectl apply -f {{ socok8s_deploy_config_location }}/armada-cluster-role-binding.yaml"
+  command: "kubectl apply -f {{ socok8s_deploy_config_location }}/armada-cluster-role-binding.yaml"
+  register: armada_cluster_role_binding_result
+  changed_when: "armada_cluster_role_binding_result.stdout|search('created')"
   tags:
     - install
+    - skip_ansible_lint
```

Comment:
is that necessary? What's the issue covered by ansible-lint?

Reply:
thx @evrardjp I should have removed 'skip_ansible_lint' tag, I am not getting any error when I remove it. Will handle this as a new PR since this PR was already merged